recipes-domx/meta-xt-images-vgpu/kernel-module-gles/kernel-module-gles_1.9.bb

```diff
@@ -0,0 +1,9 @@
+require kernel-module-gles.inc
+
+KBUILD_OUTDIR_r8a7795-es2 = "binary_r8a7795_es2_linux_${BUILD}/target_aarch64/kbuild/"
```

Comment:
I think you can move `r8a7795_es2/r8a7795/r8a7796` value to separate variable (e.g. `DDK_OUTPUT`) and define both `KBUILD_OUTDIR` and `KBUILD_DIR` using this  variable.